gandalf.js

```diff
@@ -8,40 +8,40 @@
  *
```

Comment:
SCRIPT-There seems to be a typo in the function name. Did you mean `calculateAverage` instead of `calculateAverge`?